src/main/java/org/jboss/pnc/kafkastore/common/SortedSetOrder.java

```diff
@@ -42,6 +43,7 @@
 
     ArrayList<String> sorted = new ArrayList<>();
 
+    @Timed
```

Comment:
Mmmm this is not an exposed REST enpdoint, so not sure whether @Timed is applicable. Also, not part of micrometer spec.